### PR TITLE
7.2 Developer | Correct info about -metatype in bnd files | LRDOCS-7102

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-7102, I think this is more useful for third party devs than what we had there. Ler me know if it looks good to you.

### CODE REVIEW

**Comment by rbohl** [comment]
Hi Pei-Jung, I see that your origin is way behind upstream. If you want to keep them synced without having to manually pull from upstream then push to origin, you could configure OriginAI for liferay-docs: https://grow.liferay.com/share/OriginAl

File: developer/frameworks/articles/configuration/02-creating-a-configuration-interface.markdown [hunk 1]
```diff
@@ -81,9 +79,12 @@ This automatically scopes your configuration to `SYSTEM`.
 | described below, we recommend defaulting to the bnd annotations.
 
 | **Also Note:** Your project depends on the `-metatype: *` line in it's
-| `bnd.bnd` file. This line is generated automatically from your bundle's
-| `.build.properties` file, and is required to provide information about your
-| app's configuration options so that a configuration UI can be generated.
+| `MANIFEST.MF` file. If you're in a [Liferay
```
Comment (rbohl) [inline developer/frameworks/articles/configuration/02-creating-a-configuration-interface.markdown]: @pei-jung I got this wrong. Inspecting an -api JAR file shows that this actually gets generated into a *Configuration.xml file in the module JAR's `OSGI-INF/metatype` folder. I'll update and push a commit.

**Comment by pei-jung** [comment]
> Hi Pei-Jung, I see that your origin is way behind upstream. If you want to keep them synced without having to manually pull from upstream then push to origin, you could configure OriginAI for liferay-docs: https://grow.liferay.com/share/OriginAl

Yeah, I haven't updated the repo in a long time. As you can see, I haven't touch this repo ever since I forked it, and you're the first one sending me a pull ;) And thanks for the tip. I will setup OriginAI to keep it in sync with upstream.

**Comment by pei-jung** [comment]
The changes LGTM!

**Comment by rbohl** [comment]
Forwarded: https://github.com/sez11a/liferay-docs/pull/4393
Thanks!

**Pull request closed without merging**